tips/DID_method_spec/DID_method_spec.md

```diff
@@ -0,0 +1,309 @@
+
+#### Deactivate
+
+Temporarily deactivating a DID can be done by deleting the `Metadata Entry` corresponding to the key `did:iota` in the Account Output, and publishing an [update](#update).
+
+Another option is to [update](#update) the DID Document and set the `deactivated` property in its `metadata` to true. In both cases, the deactivated DID Document will be marked as `deactivated` when resolved.
+
+Since retrieving the history of an Account to detect if an Account ever was a DID is unpractical, any Account Output lacking a `Metadata Entry` with a `did:iota` Key is considered a deactivated DID. The same applies for Account Outputs lacking the Metadata Feature.
+
+#### Destroy
+
+In order to permanently destroy a DID, a new transaction can be published that consumes the Account Output without having a corresponding Account Output on the output side with the same explicit `Account ID`. This results in destroying the Alias Output and the DID.
+
+Note that this operation irreversibly and irrecoverably deletes the DID. This is because the `Account ID` from which an IOTA DID is derived (see [IOTA-Tag](#iota-tag)) is generated from the hash of the input transaction that created it, which cannot practically be replicated.
+
+## IOTA Identity standards
+
+The `did:iota` method is implemented in the [IOTA Identity framework](https://github.com/iotaledger/identity.rs). This framework supports a number of operations that are standardized, some are standardized across the SSI community, and some are the invention of the IOTA Foundation.
+
+### Revocation
+
+Revocation of verifiable credentials and signatures can be achieved using the Revocation Bitmap 2022 where issuers store a bitmap of indices in the DID Document. These indices correspond to verifiable credentials they have issued. If the binary value of the index in the bitmap is 1 (one), the verifiable credential is revoked, if it is 0 (zero) it is not revoked.
+
+### Standardized Services
+
+The IOTA Identity framework also standardized certain services that are embedded in the DID Document. It is RECOMMENDED to implement these when implementing the did:iota method.
+
+Currently standardized services:
+
+    Revocation Bitmap Service
+
+## Migration
+
+Instructions to migrate DIDs stored on the Stardust version of the network to Nova:
+
+1. Detect all Alias Outputs containing a leading "DID" magic bytes in the ByteArray of their State Metadata.
+1. For each of those Alias Output in Stardust, create an Account Output in Nova where `Alias ID` = `Account ID`.
+1. Set a Metadata Feature in the created Account Output with an Entry where:
+   - `Key` = `did:iota`
+   - `Value` = a copy of Alias Output's State Metadata ByteArray without the leading "DID" magic bytes.
+
+If migrated Alias Output is self-governed:
+
+1. Set Account Unlock to migrated Alias Unlock
+
+Else:
+
+1. Set Account Unlock condition to MultiAddress where:
+    - migrated Alias Output state controller address is set with weight=1
```

Comment:
In this case we give the state controller more power than it had before. In the AliasOuput the state controller wasn't able to control the tokens, while now it will be able to.